homeassistant/components/camera/bomradarloop.py

```diff
@@ -0,0 +1,381 @@
+"""
+Provie animtaed GIF loops of BOM radar imagery.
+
+This component provides animated GIF images of weather-radar imagery derived
```

Comment:
This should be in the documentation.

Reply:
Would you prefer to have it removed from the source file now, or simply _moved_ to the external documentation when that's written?

Reply:
I've gone ahead and removed the extra docs and am now working toward a PR in home-assistant.io that includes them.